assets/queries/k8s/using_kubernetes_native_secret_management/metadata.json

```diff
@@ -0,0 +1,10 @@
+{
+  "id": "b9c83569-459b-4110-8f79-6305aa33cb37",
+  "queryName": "Using Kubernetes Native Secret Management",
+  "severity": "MEDIUM",
+  "category": "Secret Management",
+  "descriptionText": "Use of Kubernetes External Secret Storage and Management System should be considered if you have more complex secret management need and ensure that access to secrets is carefully limited ",
```

Comment:
```suggestion
  "descriptionText": "Kubernetes External Secret Storage and Management System usage should be considered if you have more complex secret management needs, rather than using Kubernetes Secrets directly. Additionally, ensure that access to secrets is carefully limited",
```